src/PrestaShopBundle/Controller/Admin/Sell/Order/CreditSlipController.php

```diff
@@ -27,23 +27,27 @@
 namespace PrestaShopBundle\Controller\Admin\Sell\Order;
 
 use DateTime;
+use PrestaShop\PrestaShop\Adapter\PDF\CreditSlipPdfGenerator;
 use PrestaShop\PrestaShop\Core\Domain\CreditSlip\Exception\CreditSlipNotFoundException;
 use PrestaShop\PrestaShop\Core\Domain\CreditSlip\Query\GetCreditSlipIdsByDateRange;
 use PrestaShop\PrestaShop\Core\Domain\CreditSlip\ValueObject\CreditSlipId;
 use PrestaShop\PrestaShop\Core\Exception\CoreException;
-use PrestaShop\PrestaShop\Core\Form\FormHandlerInterface;
+use PrestaShop\PrestaShop\Core\Form\Handler;
```

Comment:
Why switch this? If you can keep relying on the interface it's preferable, this way if a module overrides the implementation with something that doesn't extend our base `FormHandler` class it still works